benchmark/C/Savina/src/concurrency/ProdCons.lf

```diff
@@ -8,12 +8,10 @@
  
 target C {
     flags: "-lm",
-    logging: DEBUG,
     keepalive: true,
     timeout: 2 secs
 };
 
-import BenchmarkRunner from "../BenchmarkRunner.lf";
 
 preamble {=
     #include <stdlib.h>
```

Comment:
It seems that the whole program can be significantly simplified. See the updated version of the C++ benchmark. This version removed the various message types defined below and instead uses dedicated ports. Would be nice if we could also simplify this version.